pkg/document/json/root.go

```diff
@@ -101,6 +108,14 @@ func (r *Root) GarbageCollect(ticket *time.Ticket) int {
 		}
 	}
 
+	for _, text := range r.editedTextElementMapByCreatedAt {
+		removedNodeCnt := text.cleanupRemovedNodes(ticket)
+		if removedNodeCnt > 0 {
+			delete(r.editedTextElementMapByCreatedAt, text.CreatedAt().Key())
+		}
+		count += removedNodeCnt
+	}
+
```

Comment:
I think It would be better to process garbage collection for `Text` that has removed node rather than edited for two reasons below.

- We can reduce the size of the cache(`editedTextElementMapByCreatedAt `).
- It is more readable that GC is performed on the removed node than on the modified node.

Reply:
I changed editedTextElementMapByCreatedAt in d16bfc2 to removedNodeTextElementMapByCreatedAt.
I don't like the name because very long, but I changed. I don't come up with a better idea. 
If you have any better ideas, please share.

On the one hand, I think it would have been better if the return value of `Key()` of `CreatedAt` was defined as a type and used. In this case, I would not have had to add a suffix like `ByCreatedAt`.
Let me know what you think.